debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/PostgresConnectorIT.java

```diff
@@ -768,6 +769,26 @@ public void shouldTakeBlacklistFiltersIntoAccount() throws Exception {
         assertNoRecordsToConsume();
     }
 
+    @Test
```

Comment:
Could you add the `@FixFor` annotation?

Reply:
Could you also add another test case which ensures that a whitelisted column can be subject to column masking or truncating?

Reply:
Done, extended the test